session4/Assignment2/TestAssignment3.txt

```diff
@@ -0,0 +1,39 @@
+import static org.junit.Assert.*;
+
+import java.util.List;
+
+import org.junit.Test;
+
+public class TestAssignment3 {
+	
+/*	@Test
+	public void testcase1(){
+		String str = "ab";
+		List<String> result = Assignment3.permuteHelper(str,"");
+		String[] actuals 	= result.toArray(new String[0]);
+		String[] expecteds  = {
+				"ab",
+				"ba"
+		};
+		
+		assertArrayEquals(expecteds, actuals);
+	}*/
+	
+	@Test
+	public void testcase2(){

```

Comment:
add two more scenario in your test methods . One for ABCD and one for Wood word.